api_types.ts

```diff
@@ -370,6 +424,22 @@ export interface SyncExecutionContext extends ExecutionContext {
 
 // A mapping exists in coda that allows these to show up in the UI.
 // If adding new values here, add them to that mapping and vice versa.
+/**
+ * Special "live" date range values that can be used as the {@link defaultValue}
```

Comment:
Aside: It's odd that for all other types of `defaultValue` it just affects the display and the parameter is undefined, but in this one special case it causes the parameter to be populated with real defaults.

Reply:
Hmm yeah I agree that `defaultValue` handling is funky in general. I'm not sure this is actually different though? In both cases it populates the initial Coda formula with a value. In this case, the value populated is just dynamically evaluated based on the current date, rather than a hardcoded value. But the user will observe this in the context of a builder UI showing date range options, and doesn't have to know that under the hood it was just an initial CFL clause.